pkg/cmd/server/api/helpers.go

```diff
@@ -305,6 +305,7 @@ func GetKubeClient(kubeConfigFile string) (*kclient.Client, *restclient.Config,
 	// TODO: this should be configured by the caller, not in this method.
 	kubeConfig.QPS = 100.0
 	kubeConfig.Burst = 200
+	kubeConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
Yes, and yes.

On Fri, Jul 22, 2016 at 2:27 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/api/helpers.go
> <https://github.com/openshift/origin/pull/9814#discussion_r71923962>:
>
> > @@ -305,6 +305,7 @@ func GetKubeClient(kubeConfigFile string) (*kclient.Client, *restclient.Config,
> >  	// TODO: this should be configured by the caller, not in this method.
> >  	kubeConfig.QPS = 100.0
> >  	kubeConfig.Burst = 200
> > +	kubeConfig.ContentType = "application/vnd.kubernetes.protobuf"
>
> So on the reading side, for 1.3 we'll get?
>
>    1. empty = json
>    2. specified is respected.
>
> On the reading side for 1.4, we'll get?
>
>    1. empty = protobuf
>    2. specified is respected
>
> We only get to do one change like this. After that we'll have some files
> written with a specified ContentType.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9814/files/ef32c3c969d72c7efec9d4ee377d4d43d290d761..895443fc49722a65e8fb36a0e28540ae6f260b7a#r71923962>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_pykvjyCPaCJm2xhLPK9t0Ahuye0Nks5qYQungaJpZM4JLFc7>
> .
>